src/Adapter/MSTest.CoreAdapter/Resources/Resource.resx

```diff
@@ -172,7 +172,8 @@
     <value>{0}: {1}</value>
   </data>
   <data name="TypeLoadFailed" xml:space="preserve">
-    <value>Unable to load types from the test source '{0}'. Some or all of the tests in this source may not be discovered.</value>
+    <value>{1}
+Unable to load types from the test source '{0}'. Some or all of the tests in this source may not be discovered.</value>
```

Comment:
Put the {1} at the end.

Unable to load types from the test source '{0}'. Some or all of the tests in this source may not be discovered.
Reason : {1}

Reply:
Done. Updated the description with the current screenshot.